TPU tests fail possibly due to wrong version of torch-xla used in CI

## 🐛 Bug
*Originally reported by @ananthsub via Slack, and pasted here for tracking.*

TPU tests are failing in a pytorch version check within the XLA profiler: https://app.circleci.com/pipelines/github/PyTorchLightning/pytorch-lightning/54909/workflows/87bd6624-a3bd-47c1-ba11-43726b6969ed/jobs/153661?invite=true#step-119-6128

```
...
        if not _TORCH_GREATER_EQUAL_1_8:
>           raise MisconfigurationException("`XLAProfiler` is only supported with `torch-xla >= 1.8`")
E           pytorch_lightning.utilities.exceptions.MisconfigurationException: `XLAProfiler` is only supported with `torch-xla >= 1.8`

pytorch_lightning/profiler/xla.py:49: MisconfigurationException
=========================== short test summary info ============================
FAILED tests/profiler/test_xla_profiler.py::test_xla_profiler_instance - pyto...
======= 1 failed, 61 passed, 9 skipped, 85 warnings in 616.44s (0:10:16) =======
```

### To Reproduce

### Expected behavior
No failure.

### Environment

### Additional context


cc @tchaton @rohitgr7 @akihironitta @carmocca @borda @kaushikb11

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -116,7 +116,7 @@ jobs:
     docker:
       - image: circleci/python:3.7
     environment:
-      - XLA_VER: 1.8
+      - XLA_VER: 1.9
       - PYTHON_VER: 3.7
       - MAX_CHECKS: 240
       - CHECK_SPEEP: 5

--- a/dockers/tpu-tests/tpu_test_cases.jsonnet
+++ b/dockers/tpu-tests/tpu_test_cases.jsonnet
@@ -8,7 +8,7 @@ local tputests = base.BaseTest {
   mode: 'postsubmit',
   configMaps: [],
 
-  timeout: 900, # 15 minutes, in seconds.
+  timeout: 1200, # 20 minutes, in seconds.
 
   image: 'pytorchlightning/pytorch_lightning',
   imageTag: 'base-xla-py{PYTHON_VERSION}-torch{PYTORCH_VERSION}',

